efitools/mtoc.c

```diff
@@ -701,6 +701,8 @@ struct arch *arch)
 		}
 		else if(strcmp(sg->segname, "__DATA_CONST") == 0)
 		    nscns++;
+		else if(strcmp(sg->segname, "__SBAT") == 0)
+    	    nscns++;
```

Comment:
This looks misaligned, did you use spaces for indentation instead of tabs?

Reply:
I copied the existing indentation from the exiting similar line above.
Looks aligned here and in my IDE with this (usual process I follow for Acidanthera stuff) but perhaps being misled.
I will check in Xcode as my IDE does default to tabs.

Reply:
It is a weird mixture of tabs and spaces, fixed it manually!